app/assets/stylesheets/global.sass

```diff
@@ -237,18 +248,93 @@ nav.navbar.navbar-default
           display: initial
 
 
-.base-navbar
+.base-navbar, nav.navbar.navbar-default .simple-navbar
+  @media (max-width: $screen-xs-max)
+    max-height: 100vh
+    overflow-y: scroll
+
   .search
     opacity: 0
     +transition(all 0.3s ease)
     a
       cursor: default
+
+  .dropdown-menu
+    border: none
+
+  .dropup .dropdown-menu, .navbar-fixed-bottom .dropdown .dropdown-menu
+    margin-bottom: 0
+
+  .nav.navbar-nav
+    .dropdown.open
+      a
+        background: linear-gradient(90deg, rgba(17, 17, 17, 0.7) 0, rgba(17, 17, 17, 0.7) 100%), linear-gradient(90deg, rgba(81, 51, 147, 0.85) 0, rgba(81, 51, 147, 0.85) 100%), #fff
```

Comment:
When your gradient has only two colors and they stand in 0% and 100% you can omit their position:
`background: linear-gradient(90deg, rgba(17, 17, 17, 0.7), rgba(17, 17, 17, 0.7)), linear-gradient(90deg, rgba(81, 51, 147, 0.85), rgba(81, 51, 147, 0.85)), #fff`